### PR TITLE
Update CH > quitMarket

### PR DESCRIPTION
**Pull request by DylanYang0523**
### PR Reminders

Be ware of the followings

- [ ] implement deployment script for your changes in deployment repo
- [ ] add verification in hardhat simulation(000-prepare-simulation-check, 902-newMarket-check or 903-simulation-check) and system test(901-system-test) in deployment repo
- [ ] update change log
 
---

1. **Contract**: make sure the code follows our convention, ex: naming, explicit returns, etc.; if uncertain, discuss with others

2. **Test**: make sure tests can cover most normal and edge cases; if not, open follow-up tickets. Also, look out for failed tests and fix them!

3. **CHANGELOG.md**: update when external interfaces are changed

4. Workflow: 
    - Github: assign the pr to yourself; if pairing, can merge directly; else, assign someone to help review
    - Asana: assign the corresponding ticket to yourself and leave the pr link on it for easier follow-ups
    - Discord: if someone is mentioned in this pr, tag on Discord

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203318394139422
  - https://app.asana.com/0/0/1203318394139427

### CODE REVIEW

File: contracts/ClearingHouse.sol [hunk 1]
```diff
@@ -1104,12 +1129,16 @@ contract ClearingHouse is
...
 
     function _getTakerPositionSafe(address trader, address baseToken) internal view returns (int256) {
-        int256 takerPositionSize = IAccountBalance(_accountBalance).getTakerPositionSize(trader, baseToken);
+        int256 takerPositionSize = _getTakerPosition(trader, baseToken);
         // CH_PSZ: position size is zero
         require(takerPositionSize != 0, "CH_PSZ");
         return takerPositionSize;
     }
 
+    function _getTakerPosition(address trader, address baseToken) internal view returns (int256) {
```
Comment (Detoo) [inline contracts/ClearingHouse.sol:1138]: suggest renaming to `_getTakerPositionSize()`

**Review by Detoo — APPROVED** [review]
lgtm except cosmetic suggestions

File: contracts/ClearingHouse.sol (hunk 1 above)
Comment (DylanYang0523) [reply contracts/ClearingHouse.sol:1138]: we want to follow the naming like `_getTakerPositionSafe` function, in this case, we don't want to update `_getTakerPositionSafe` to `_getTakerPositionSizeSafe`, so we would suggest keeping the naming same, what do you think?

Comment (DylanYang0523) [reply contracts/ClearingHouse.sol:1138]: line 1131 > _getTakerPositionSafe

Comment (Detoo) [reply contracts/ClearingHouse.sol:1138]: ah ok, didn't realize there's already another function doing this. Fine then
